submissions/MosQuitO404/js-dom/index.js

```diff
@@ -0,0 +1,76 @@
+import { teamsData } from "./data.js";
+
+const hamb = document.querySelector('#hamb');
+const popup = document.querySelector('#popup');
+const menu = document.querySelector('#menu');
+const popupMenu = document.querySelector('#menu').cloneNode(1);
```

Comment:
Why do we need `menu` and `popupMenu`?

Reply:
Popup menu is necessary for responsive design.

`const popupMenu = document.querySelector('#menu').cloneNode(1);

function renderPopup() {
    popup.append(popupMenu);
}`

Reply:
Yeah, but if you (at [this line](https://github.com/kottans/frontend-2022-homeworks/pull/397/files#diff-9418cbc5693e850ce6d4024b491585557996a4a20e96c0162c300b1954f075daR21)) replace `popupMenu` to `menu` you can drop related to `popupMenu` logic

Reply:
@lysenko-sergey-developer Yes, but when we return from mobile version to pc, without refreshing page, `menu` is gone, because we `append` it from `body` to `popup.` Like this: 
![image](https://user-images.githubusercontent.com/108759325/192104955-8ec768dc-de12-449f-94c5-7938ff5e4146.png)
I've tried to `insertAdjacentElement` `menu` to `body` again when `popup` closed and I did it, but it is not working when `popup` is open (we see `menu`) in mobile version and at the same time we return to the pc version of the site           
Cloned menu, resolving this problem.